### PR TITLE
JP-3096: Add new JWST keywords to track CR's detected by the jump step

### PR DESCRIPTION
**Pull request by mwregan2**
Resolves [JP-3096](https://jira.stsci.edu/browse/JP-3096)

This PR adds two FITS header keywords that track the rate of regular cosmic rays and shower/snowballs.

**Checklist**
- [x] added entry in `CHANGES.rst` (either in `Bug Fixes` or `Changes to API`)
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [x] updated relevant milestone(s)
- [x] added relevant label(s)

### CODE REVIEW

File: src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml [hunk 1]
```diff
@@ -927,6 +927,14 @@ properties:
             type: boolean
             fits_keyword: SELFREF
             blend_table: True
+          primary_cosmic_rays:
+            title: Rate of Cosmic Rays per thousand pixels per second
```
Comment (hbushouse) [inline src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml:931]:
```suggestion
            title: Cosmic Ray rate per thousand pixels per second
```
Original title string is too long to fit within FITS rec without being truncated.

**Review by hbushouse — COMMENTED** [review]
Needs a change log entry.

**Review by hbushouse — CHANGES_REQUESTED** [review]
Still needs a change log entry so that we can merge. @mwregan2

**Comment by mwregan2** [comment]
I updated changes.rst in stdatamodels


From: Howard Bushouse ***@***.***>
Reply-To: spacetelescope/stdatamodels ***@***.***>
Date: Friday, June 23, 2023 at 11:29 AM
To: spacetelescope/stdatamodels ***@***.***>
Cc: Michael Regan ***@***.***>, Mention ***@***.***>
Subject: Re: [spacetelescope/stdatamodels] JP-3096: Add new JWST keywords to track CR's detected by the jump step (PR #173)


@hbushouse requested changes on this pull request.

Still needs a change log entry so that we can merge. @mwregan2<https://urldefense.com/v3/__https:/github.com/mwregan2__;!!CrWY41Z8OgsX0i-WU-0LuAcUu2o!1kwcpJ5I1jTdww3sewjU_GL3rw85zH3MplXCjVb2GPmq1_945mR6gfQOKb9yLppru54FXlg_IJhFoo81sERZ_VY$>

File: CHANGES.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@
 1.6.0 (2023-06-15)
 ==================
 
+Core.schema.yaml
+----------------
+- Added two new header keywords to the Exposure section: PRIMECRS and
+  EXTNCRS [#173]. The track the rate of primary cosmic rays and extended cosmic
+  rays (Snowballs and Showers)
```
Comment (hbushouse) [inline CHANGES.rst:13]:
```suggestion
jwst.datamodels
---------------
- Added two new header keywords to the JWST exposure section: PRIMECRS and
  EXTNCRS, which are used to record the rate of primary cosmic rays and extended cosmic
  rays (Snowballs and Showers). [#173]
```

**Review by hbushouse — APPROVED** [review]

**Pull request merged**